### PR TITLE
feat/36: Style for people in groups

### LINKED ISSUE #36: [Participants list page]: Style for people in groups

### PR DESCRIPTION
**Pull request by ribeiroevandro**
Resolves #36

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -36,5 +36,5 @@ yarn-error.log*
 next-env.d.ts
 
 # sqlite
-prisma/dev.db*
+prisma/db.dev
```
Comment (fdaciuk) [inline .gitignore]: You don't need to update this. The error is in `.env.example` file: the proper name is `dev.db` tho.

Please, use `dev.db` in your `.env` file and undo this change.

**Review by fdaciuk — CHANGES_REQUESTED** [review]
Thanks @ribeiroevandro ❤️ 
Just left one comment, and we're ready to go \o/

File: .gitignore (hunk 1 above)
Comment (ribeiroevandro) [reply .gitignore]: Ok, I'll make the correction and I missed it too, to mark the winner of each round.

**Review by fdaciuk — APPROVED** [review]
